### PR TITLE
Login and Signup screen

### PR DESCRIPTION
**Pull request by manojncsoni007**
Screenshot (180)
Screenshot (179)

### CODE REVIEW

**Review by aurobindoGupta — APPROVED** [review]
looks Good!

File: pages/authentication/login.html [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+                <label for="password" class="label">Password *</label>
+                <input type="password" id="password" class="input" placeholder="Enter password">
+            </div>
+            <div class="login-remember">
+                <label for="remember-me"><input type="checkbox" id="remember-me">Remember Me</label>
+                <a href="" class="forgot-password">Forgot your password ?</a>
+            </div>
+            <button class="authentication-btn">Login</button>
+            <div class="flex-center">
+                <a href="./signup.html"> Creat New Account <i class="fa fa-chevron-right"></i></a>
```
Comment (prakashsakari) [inline pages/authentication/login.html:62]: Spell check - Create New Account

**Review by HeyNitin — APPROVED** [review]
LGTM

**Review by prakashsakari — APPROVED** [review]
Added a comment for spell check, rest looks good to me

File: style.css [hunk 2]
```diff
@@ -5,6 +5,6 @@
 }
 
 .search-bar input {
-    width: 15rem;
+    width: 25rem;
```
Comment (starver20) [inline style.css:8]: You can create css variables for spacings too.

File: pages/authentication/auth.css [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+.forgot-password {
+    font-size: 0.9rem;
+    float: right;
+}
+
+.authentication-btn {
+    display: block;
+    width: 100%;
+    margin: 1rem 0;
+    padding: 0.5rem;
```
Comment (starver20) [inline pages/authentication/auth.css:28]: You can create css variables for spacing, so that it remains consistent everywhere.

**Pull request merged**